common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationRequest.java

```diff
@@ -106,17 +130,24 @@ public void setState(final String state) {
         mState = state;
     }
 
+    public Set<String> getScope() {
+        return mScope;
+    }
+
+    public void setScope(final Set<String> scope) {
+        mScope = new HashSet<>(scope);
+    }
+
     //CHECKSTYLE:OFF
     @Override
     public String toString() {
         return "AuthorizationRequest{" +
                 "mResponseType='" + mResponseType + '\'' +
                 ", mClientId='" + mClientId + '\'' +
                 ", mRedirectUri='" + mRedirectUri + '\'' +
-                ", mScope='" + mScope + '\'' +
+                ", mScope=" + StringExtensions.convertSetToString(mScope, " ") + '\'' +
```

Comment:
Hand modifying autogenerated methods feels like an opportunity to create an easy regression point. Is this `toString()` needed? If yes, why? If not, let's abandon.

(If we are reliant on the specific behavior of a `toString()` we should provide a unit test for it....)